src/main/java/server/entities/Facility.java

```diff
@@ -19,45 +18,37 @@ public Facility(String name){
     public void addBooking(Booking booking){
 
         //Check
-        LocalTime startTime = booking.getStart().toLocalTime();
-        LocalTime endTime = booking.getEnd().toLocalTime();
-        LocalDate startDate =booking.getStart().toLocalDate();
-        LocalDate endDate = booking.getEnd().toLocalDate();
-        //TODO should thre be a
+        LocalTime startTime = booking.getStart();
+        LocalTime endTime = booking.getEnd();
+        String day =booking.getDay();
+
         if((startTime.compareTo(BookingManager.getOpenTime()) <0) || endTime.compareTo(BookingManager.getCloseTime()) > 0){
             System.out.println("Booking invalid: start time is too early, or end time is too late");
             return;
         }
```

Comment:
suggest to split into two clauses, so the client knows exactly what is the problem (too early or too late)

Reply:
good point, i will update this

Reply:
edited such that Facility functions (addBooking and removeBooking) will not return exceptions. BookingManager will manage all exceptions created and returned. addBooking and removeBooking, will be put in a try catch block instead to catch unexpected errors, but this catch block should never be reached bcos BookingManager should have done the checks